.github/workflows/ci.yml

```
@@ -366,3 +366,36 @@ jobs:
             -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }} `
             -DCMAKE_C_COMPILER=${{ matrix.c_compiler }} -DTBB_CPF=${{ matrix.preview }} -DTBB_TEST=OFF -DTBB_EXAMPLES=ON ..
           cmake --build . -v --target light_test_examples
+
+  linux-doc-examples-testing:
+    name: doc_examples_${{ matrix.os }}_${{ matrix.cxx_compiler }}_cxx${{ matrix.std }}_${{ matrix.build_type }}
+    runs-on: ['${{ matrix.os }}']
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            c_compiler: gcc
+            cxx_compiler: g++
+            std: 11
+            build_type: relwithdebinfo
+            job_name: linux_doc_examples_testing_ubuntu_latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build library
+        run: |
+          mkdir build && cd build
+          cmake -DCMAKE_CXX_STANDARD=${{ matrix.std }} -DCMAKE_BUILD_TYPE=${{ matrix.build_type}} \
+            -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }} -DCMAKE_C_COMPILER=${{ matrix.c_compiler }} \
+            -DTBB_TEST=OFF ..
+          cmake --build . -v --target tbb --target tbbmalloc --target tbbmalloc_proxy
+      - name: Test doc examples
+        run: |
+          source `find ./build -name vars.sh`
+          cd doc/main/examples_testing
+          mkdir build && cd build
+          cmake -DCMAKE_CXX_STANDARD=${{ matrix.std }} -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
+            -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }} -DCMAKE_C_COMPILER=${{ matrix.c_compiler }} ..
+          cmake --build . -v --target build-doc-examples
```

If we require CMake of version 3.12 at least, we will be able to build in parallel. 
```suggestion
          cmake --build . --parallel -v --target build-doc-examples
```

I suppose `ubuntu-latest` is the latest available version of Ubuntu Linux. Even if it is the LTS release, which is currently 24.4, `cmake --version` reports "cmake version 3.28.3" on it. So, I think it is more than enough to be able to build in parallel.

---

Well, it is Linux. We can just call `VERBOSE=1 make -j$(nproc) build-doc-examples`

---

I would prefer writing the code that has fewer platform-specific dependencies. So, if it is possible, it is better to use `cmake` build command. Especially, if it can be then easily adjusted on the other platforms.

---

And why make it deliberately even more platform-specific? Especially, when likely there is a need running this in other platforms.

I guess we have expressed our viewpoints. Let's then just the author of the patch decide on this.